### ISSUE TITLE
Zoechip3.org there are still ads on it and it opens pop ups when you click on the site

### ISSUE THREAD

**Issue by olstice** [AdguardTeam/AdguardFilters#151149]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for iOS

### AdGuard version

4.4.6

### What type of problem have you encountered?

Missed ads or ad leftovers, Annoyances — pop-ups, cookie warnings, etc.

### Which browser(s) do you use?

Safari

### Which device do you use?

Mobile

### Where is the problem encountered?

https://www3.zoechip.com/movie/spirited-90331

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter

### Add your comment and screenshots

1. <comment> Hi so im going on zoechip3.org to try and watch a movie and even with my adguard up to date, latest version, and Mobil ads filter and base filter it still has ads on the site that i annoyingly have to keep closing only to open pop us which i have to close of those and if you click on the site it opens pop up tabs 

you literally have to face a couple pop up tabs to finally be able to use the site any help to this would be very appreciative 
3. Screenshots
<details><summary>Screenshot 1:</summary> 

]

</details>

<details><summary>Screenshot N:</summary
![95476ADB-CCD6-4D17-83F5-5CEE56473FD7](https://github.com/AdguardTeam/AdguardFilters/assets/124982114/24651f2c-81de-49ea-9afa-de81ac4976a1)
>

</details>

### Privacy

- [X] I agree to follow this condition

**Comment by krystian3w** [comment]
Try force update Filters and wait 60 second after process end (disappear all gray bars).

If it still doesn't work, you/we probably need to come up with rules to hide the leak.

**Comment by olstice** [comment]
> Try force update Filters and wait 60 second after process end (disappear all gray bars).
> 
> If it still doesn't work, you/we probably need to come up with rules to hide the leak.

It worked thank you! Did you guys do updates to the Mobil and base filters?

**Comment by krystian3w** [comment]
Base Filter is updated daily: https://github.com/AdguardTeam/AdguardFilters/commits/master/BaseFilter/sections

Mobil too, but with less commits: https://github.com/AdguardTeam/AdguardFilters/tree/master/MobileFilter/sections

I'm unlikely to check if any new/fresh generic rule(s) or one quietly added blocks these ads.

Still to be checked are these commits from the last two days: https://github.com/easylist/easylist/commits/master/easylist

as the EasyList rules are an integral part of the "AdGuard Base" Filter.

---

The most likely seems to be a known problem with Filters processing when an application is suddenly closed or a background execution terminates badly.

**Comment by olstice** [comment]
> Try force update Filters and wait 60 second after process end (disappear all gray bars).
> 
> If it still doesn't work, you/we probably need to come up with rules to hide the leak.


    So I thought I had this resolved turns out it was only a temporary solution the website Zoechip has an imbeded url link that contains the exact javascript code that causes the malicious redirects of which right now its https://zkwl.zwmvyzyrlvzew.top/kbwaazlzkwjla/abqaj

before that it was https://krze.klrqqylrzzzj.top/kzzeenqvnqjww/abqaj

the only connection both url have is the .top/abqaj everything else is different.

And no matter which one i block of these rules ||zwmvyzyrlvzew.top^$important klrqqylrzzzj.top^$important ||qnlranvabrnq.top^$important

A new one just gets added into the head meta tag section of the html making the rule no longer apply

I need help please!!!!





I checked both javascripts on diffcheck they are both 100% identical expect for this

this script url link https://krze.klrqqylrzzzj.top/kzzeenqvnqjww/abqaj

has this added of text the second url link does not

be('{"7tw":"c99uf:\/\/t7jto5mbmwmc.b7t7\/oUVlirBdWldzWVwRRBc3ZaR\/e6ge8\/?f5la9rk9_t=jzHNpJjMRY18SOfqUOinmxmPNuA8Yh6Zt1aTZKrkS4E&atj=6&umtmx_e=alt9j_nmwwjm53&xz=[xzbwc]&r5=[r5]","vlar_oz":e6ge8,"ulu_9iur":"9mj7u","ntrh7ra5i":q,"5muuoab":840,"rdrti_umbr":nmwfr,"rdrti_dor2":nmwfr,"rdrti_dofo9":9t7r,"l9crt_5wo53_on_5wlfr":9t7r,"zrwmi_jrnltr_f9mt9_fr5lazf":0,"zrwmi_jrnltr_f9mt9_5wo53f":0,"oa9rtdmw_jr92rra_mzf_fr5lazf":60,"oa9rtdmw_jr92rra_mzf_5wo53f":0,"5ff_oa5w7zr":[],"5ff_rk5w7zr":[".bwk-2m9rtxmt3",".bwk-2m9rtxmt3-utrdor2-5wlfr",".bwk-2m9rtxmt3-utrdor2-zm9m",".bwk-2m9rtxmt3-utrdor2-5la9moart"],"mzjwl53_fcl2":9t7r,"ht_vlar_oz":0,"ucowma9ctluo5_wrdrw":0,"mj_frtdrtf_7tw":"","x7tx7t":"","9oxrvlar_lnnfr9":-q,"rk9razrz_vlar":nmwfr,"obaltr_9oxrvlar_5cr53":nmwfr,"zrd9llwf_utl9r59ola":9t7r,"9oxrvlar_zonn":10,"zofmjwr_xmoa_umbr":nmwfr,"zofmjwr_m79l_uluf":nmwfr,"9tm5r":0,"buu":nmwfr,"zofmjwr_rxu9i_umbr_5cr53":nmwfr,"utrnr95c_9oxrl79":q400,"f5tlww_utl9r59ola":9t7r,"xr9to5_7tw":"c99uf:\/\/3tvr.3wthhiwtvvvs.9lu\/x9a\/e6ge8\/0qre50r01ejny1p4rnrp1rn65yznye5g.y46yy4gqg6.6qq","77oz_7tw":"c99uf:\/\/b7wfm5cuitrkom.7al\/57oz\/","nmww_trntrfc_7tw":"c99uf:\/\/7w7w7j7bwrf.2rjfo9r\/tn\/e6ge8","tl9_7tw":"c99uf:\/\/futo9vm2mu7co.b7t7\/bz\/e6ge8?xz=[xzbwc]"}','abcdefghijklmnopqrstuvwxyz0123456789mj5zrnbcos3wxaluhtf97d2kiv068e4q1gpy',{withUserId:!0,withLogger:!0,withTimeZoneCheck:!0,withDevtools:!0,withStrangeScrollObserver:!0,metricType:K});if(!t)return;const{settings:e,log:n,metric:i}=t,{[Je]:s,[Fe]:o,[ne]:r,[Re]:c,[Ue]:u,[Be]:d,[Ce]:h,[We]:w,[on]:f,[ce]:p}=e;o&&'/'===location.pathname||Se((()=>{if(f){const

And this url link https://zkwl.zwmvyzyrlvzew.top/kazbblrlvkajw/abqaj

has this addition replaceing the the first url links one

be('{"7nq":"krr9y:\/\/n7bnmuocoqok.c7n7\/m33FwyglHDYeoQ4OCXjA\/0h80a\/?yu1er2ir_n=SGnXnNHIK_Lb1HRS7dU1F4&enb=h&9onog_0=e1nrb_loqqbouz&gj=[gjcqk]&2u=[2u]","x1e2_mj":0h80a,"919_rv92":"rob79","ln2w72euv":5,"uo99mec":afp,"2d2nv_9oc2":loqy2,"2d2nv_dm24":loqy2,"2d2nv_dmymr":rn72,"1rk2n_uqmuz_ml_uq1y2":rn72,"j2qov_b2l1n2_yronr_y2u1ejy":p,"j2qov_b2l1n2_yronr_uqmuzy":p,"mer2ndoq_b2r422e_ojy_y2u1ejy":hp,"mer2ndoq_b2r422e_ojy_uqmuzy":p,"uyy_meuq7j2":[],"uyy_2iuq7j2":[".cqi-4or2ngonz",".cqi-4or2ngonz-9n2dm24-uq1y2",".cqi-4or2ngonz-9n2dm24-joro",".cqi-4or2ngonz-9n2dm24-u1erome2n"],"ojbq1uz_yk14":rn72,"wn_x1e2_mj":p,"9kmqoerkn19mu_q2d2q":p,"ob_y2nd2ny_7nq":"","g7ng7n":"","rmg2x1e2_1lly2r":-5,"2ir2ej2j_x1e2":loqy2,"mce1n2_rmg2x1e2_uk2uz":loqy2,"j2dr11qy_9n1r2urm1e":rn72,"rmg2x1e2_jmll":6p,"jmyobq2_gome_9oc2":loqy2,"jmyobq2_o7r1_919y":loqy2,"rnou2":p,"c99":loqy2,"jmyobq2_2g9rv_9oc2_uk2uz":loqy2,"9n2l2ruk_rmg217r":5fpp,"yun1qq_9n1r2urm1e":rn72,"g2rnmu_7nq":"krr9y:\/\/xz4q.x4gdvxvnqdx24.r19\/gre\/0h80a\/aj8paf82abj0ojhuh2fjjphp2fhoop6h.tfhttf6s50.8t0","77mj_7nq":"krr9y:\/\/c7qyouk9vn2imo.7e1\/u7mj\/","loqq_n2ln2yk_7nq":"krr9y:\/\/7q7q7b7cq2y.42bymr2\/nl\/0h80a","n1r_7nq":"krr9y:\/\/y9nmrxo4o97km.c7n7\/cj\/0h80a?gj=[gjcqk]"}','abcdefghijklmnopqrstuvwxyz0123456789obuj2lckm3zqge19wnyr7d4ivxpha0f568st',{withUserId:!0,withLogger:!0,withTimeZoneCheck:!0,withDevtools:!0,withStrangeScrollObserver:!0,metricType:K});if(!t)return;const{settings:e,log:n,metric:i}=t,{[Je]:s,[Fe]:o,[ne]:r,[Re]:c,[Ue]:u,[Be]:d,[Ce]:h,[We]:w,[on]:f,[ce]:p}=e;o&&'/'===location.pathname||Se((()=>{if(f){const

Other than that they are 100% identical javascripts

**Comment by Alex-302** [comment]
@olstice They use inline initial script. Sometimes it cannot be blocked stable on iOS. iOS does not have native support of advanced syntax, and our implementation is executed with a delay (~200 ms).

**Comment by olstice** [comment]
> @olstice They use inline initial script. Sometimes it cannot be blocked stable on iOS. iOS does not have native support of advanced syntax, and our implementation is executed with a delay (~200 ms).

actually it wasnt stable blocked on windows either and i believe its because i was blocking the exact url link that contains the javascript of the redirect but sadly everynow and than the url link changes so the userrule i added no longer applies?

**Comment by olstice** [comment]
> @olstice They use inline initial script. Sometimes it cannot be blocked stable on iOS. iOS does not have native support of advanced syntax, and our implementation is executed with a delay (~200 ms).

Also what is inline initial script im actually wanting to learn this stuff so i dont always have to bother you guys and so i can be more indepedent thats why i was able to find the culprit url in the first place im trying to learn!

**Comment by olstice** [comment]
> @olstice They use inline initial script. Sometimes it cannot be blocked stable on iOS. iOS does not have native support of advanced syntax, and our implementation is executed with a delay (~200 ms).

its not fixed! i updated by adguard and if you click anywhere of the website https://www3.zoechip.com/

it still redirects you to another page

The culprit is this link https://kayb.krvemwbmjzrwy.top/kqmlvellejwmj/abqaj

problem is this link located in the script tag of the head tag of html which contains the javascript code of the redirect changes freqently to a different url link making the initnial userule to the last url obsolute

before that it was this url link https://krze.klrqqylrzzzj.top/kzzeenqvnqjww/abqaj

it changes constanly with the only reconsiable part of every url being the .top and the /abqag

of which each ones code of every url link is of the majority the exact with a bit added that was not in the last

**Comment by krystian3w** [comment]
try check:

 `view-source:https://www3.zoechip.com/`
 `view-source:http://zoechip.com/`

 maybe their exist latest inline redirect script.

You can see if this manages to block line scripts on the home page in 200 ms:

```adb
||zoechip.com/|$inline-script,important
```

With that said, I don't keep track of whether inline-script has been added for iOS, then may works only:

```adb
||zoechip.com/|$csp=script-src 'self' 'unsafe-eval' http: https: data: blob: mediastream: filesystem:,important
```

if `csp=` works on iOS.

As the site by IP injects inline JS, you can always open its source code at first and only then fire it normally.

If page "bake" cookies/localStorage, then maybe use this as homepage: https://fiddle.jshell.net/9ngLtwx5/2/show/dark/

**Comment by olstice** [comment]
> try check:
> 
> `view-source:https://www3.zoechip.com/` `view-source:http://zoechip.com/`
> 
> maybe their exist latest inline redirect script.
> 
> You can see if this manages to block line scripts on the home page after 200 ms delay:
> 
> ```
> ||zoechip.com/|$inline-script,important
> ```
> 
> With that said, I don't keep track of whether inline-script has been added for iOS, then may works only:
> 
> ```
> ||zoechip.com/|$csp=script-src 'self' 'unsafe-eval' http: https: data: blob: mediastream: filesystem:,important
> ```
> 
> if `csp=` works on iOS.
> 
> As the site by IP injects inline JS, you can always open its source code at first and only then fire it normally.
> 
> If page "bake" cookies/localStorage, then maybe use this as homepage: https://fiddle.jshell.net/9ngLtwx5/2/show/dark/ = simple adware should not go back/outside at used iframe/object/embed.

How do you guys figure out these userrules? i actually want to know i want to learn!

**Comment by olstice** [comment]
> try check:
> 
> `view-source:https://www3.zoechip.com/` `view-source:http://zoechip.com/`
> 
> maybe their exist latest inline redirect script.
> 
> You can see if this manages to block line scripts on the home page after 200 ms delay:
> 
> ```
> ||zoechip.com/|$inline-script,important
> ```
> 
> With that said, I don't keep track of whether inline-script has been added for iOS, then may works only:
> 
> ```
> ||zoechip.com/|$csp=script-src 'self' 'unsafe-eval' http: https: data: blob: mediastream: filesystem:,important
> ```
> 
> if `csp=` works on iOS.
> 
> As the site by IP injects inline JS, you can always open its source code at first and only then fire it normally.
> 
> If page "bake" cookies/localStorage, then maybe use this as homepage: https://fiddle.jshell.net/9ngLtwx5/2/show/dark/ = simple adware should not go back/outside at used iframe/object/embed.



    The inline script userule suggested to me did work for windows adguard extension ||zoechip.com/|$inline-script,important

    it however did not work on Adguard IOS

    of which neither did this userule work on IOS either

    ||zoechip.com/|$csp=script-src 'self' 'unsafe-eval' http: https: data: blob: mediastream: filesystem:,important

    So for windows the redirect is blocked for IOS its not

**Comment by olstice** [comment]
> try check:
> 
> `view-source:https://www3.zoechip.com/` `view-source:http://zoechip.com/`
> 
> maybe their exist latest inline redirect script.
> 
> You can see if this manages to block line scripts on the home page after 200 ms delay:
> 
> ```
> ||zoechip.com/|$inline-script,important
> ```
> 
> With that said, I don't keep track of whether inline-script has been added for iOS, then may works only:
> 
> ```
> ||zoechip.com/|$csp=script-src 'self' 'unsafe-eval' http: https: data: blob: mediastream: filesystem:,important
> ```
> 
> if `csp=` works on iOS.
> 
> As the site by IP injects inline JS, you can always open its source code at first and only then fire it normally.
> 
> If page "bake" cookies/localStorage, then maybe use this as homepage: https://fiddle.jshell.net/9ngLtwx5/2/show/dark/ = simple adware should not go back/outside at used iframe/object/embed.



    The inline script userule suggested to me did work for windows adguard extension ||zoechip.com/|$inline-script,important

    it however did not work on Adguard IOS

    of which neither did this userule work on IOS either

    ||zoechip.com/|$csp=script-src 'self' 'unsafe-eval' http: https: data: blob: mediastream: filesystem:,important

    So for windows the redirect is blocked for IOS its not

**Comment by olstice** [comment]
> try check:
> 
> `view-source:https://www3.zoechip.com/` `view-source:http://zoechip.com/`
> 
> maybe their exist latest inline redirect script.
> 
> You can see if this manages to block line scripts on the home page after 200 ms delay:
> 
> ```
> ||zoechip.com/|$inline-script,important
> ```
> 
> With that said, I don't keep track of whether inline-script has been added for iOS, then may works only:
> 
> ```
> ||zoechip.com/|$csp=script-src 'self' 'unsafe-eval' http: https: data: blob: mediastream: filesystem:,important
> ```
> 
> if `csp=` works on iOS.
> 
> As the site by IP injects inline JS, you can always open its source code at first and only then fire it normally.
> 
> If page "bake" cookies/localStorage, then maybe use this as homepage: https://fiddle.jshell.net/9ngLtwx5/2/show/dark/ = simple adware should not go back/outside at used iframe/object/embed.

I dont understand anything your saying i want to i want to learn i strive to learn but maybe im just too basic for this maybe im too dumb

**Comment by olstice** [comment]
> @olstice They use inline initial script. Sometimes it cannot be blocked stable on iOS. iOS does not have native support of advanced syntax, and our implementation is executed with a delay (~200 ms).

wait now im confused i looked it up and an inline script does not include the src= right?

but this script does include the src= which i believe i remember is the sourse so its an external javascript file right?

**Comment by krystian3w** [comment]
There is nothing to understand there for now, just into the address bar paste one of the two:

`view-source:https://www3.zoechip.com/`
`view-source:http://zoechip.com/`

 and just start watching videos for "free" with open page now normally. The method will only work if they check by the IP of your network connection if this is your first visit to the site. You can try to check over Wi-Fi as you view this site more often on a "macbook"/iMac.

---
---

As the site needs to "bake" the "you've seen our ugly ads" cake, it's better to open it from a substitute such as: https://fiddle.jshell.net/9ngLtwx5/2/show/dark/

**Comment by krystian3w** [comment]
> but this script does include the src= which i believe i remember is the sourse so its an external javascript file right?

Yes.

https://adguard.com/kb/general/ad-filtering/create-own-filters/#basic-rules-special-characters - maybe possible use wildcards if some letters/numers are stable in `src="..."`.

If nope then needed is learn regex and what regex is now suported at iOS:

https://adguard.com/kb/general/ad-filtering/create-own-filters/#regexp-support

**Comment by olstice** [comment]
> view-source:https://www3.zoechip.com/



all this did is showed the inspect element html of the website without me needing to go into inspect element

**Comment by olstice** [comment]
> > but this script does include the src= which i believe i remember is the sourse so its an external javascript file right?
> 
> Yes.
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#basic-rules-special-characters - maybe possible use wildcards if some letters/numers are stable in `src="..."`.
> 
> If nope then needed is learn regex and what regex is now suported at iOS:
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#regexp-support

Why is everytime i report miss ads on the adguard website and create a ticket that ticket is just added into the previous ones i also made on the same thread?

**Comment by olstice** [comment]
> > but this script does include the src= which i believe i remember is the sourse so its an external javascript file right?
> 
> Yes.
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#basic-rules-special-characters - maybe possible use wildcards if some letters/numers are stable in `src="..."`.
> 
> If nope then needed is learn regex and what regex is now suported at iOS:
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#regexp-support

and it saids safari cannot open page for its invalid on my iphone


Also this link you keep sending

**Comment by olstice** [comment]
> > but this script does include the src= which i believe i remember is the sourse so its an external javascript file right?
> 
> Yes.
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#basic-rules-special-characters - maybe possible use wildcards if some letters/numers are stable in `src="..."`.
> 
> If nope then needed is learn regex and what regex is now suported at iOS:
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#regexp-support

Krystian I dont know how to create my own filter rules i dont understand any of this i tried to and i dont get it so i need someone to do this for me, because im not capable

**Comment by olstice** [comment]
> > but this script does include the src= which i believe i remember is the sourse so its an external javascript file right?
> 
> Yes.
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#basic-rules-special-characters - maybe possible use wildcards if some letters/numers are stable in `src="..."`.
> 
> If nope then needed is learn regex and what regex is now suported at iOS:
> 
> https://adguard.com/kb/general/ad-filtering/create-own-filters/#regexp-support

![image](https://github.com/AdguardTeam/AdguardFilters/assets/124982114/9f08900d-facf-4e6d-adf4-ea321377d26d)


i dont comprehend this at all

i mean i guess 

    http://example.org/ad1.gif
    http://subdomain.example.org/ad1.gif
    https://ads.example.org:8000/
    
    
    All these blook because its first the address of the website instead of a part of the website and then the address of the website?
    
    
    
    and i guess these dont block because 
    
    
        http://ads.example.org.us/ad1.gif
    http://example.com/redirect/http://ads.example.org/
    
    
    Because its the part of the website you want to block being listed first before the address of the website? and the second one isnt blocking because while it is the address of the website listed first, with then somewhat of the website than added, its than also  with something else also added of the website which is fine but the address of the website added again at the end?
    
    Where i suppose it should be 
    
    
    http//example.org/us/ad1.gif      and the second one should be this?
    
    
    http://example.org/redirect/ads
    
    
    if this isnt correct than i have no idea why one list is blocked but other is not?

**Comment by krystian3w** [comment]
> all this did is showed the inspect element html of the website without me needing to go into inspect element

There, to me, it's just about fooling the server that you are visiting a site for the first time on a given IP (some fraction of malware checks the IP to avoid loading a second time on the site).

---

> Krystian I dont know how to create my own filter rules i dont understand any of this i tried to and i dont get it so i need someone to do this for me, because im not capable

OK, if you have the desire you can try to learn what a wildcard is from other sources such as Adblock Plus specifications:

https://forum.adblockplus.org/viewtopic.php?t=54688

https://help.adblockplus.org/hc/en-us/articles/360062733293#anchors

"In short, an asterisk in a simple web filter replaces 1 or many characters - they allow any combination."

Mastering regexes rather well start with some beginner's courses and perhaps you will refresh it to yourself throughout your life e.g. problems with writing regexes so that it is not too loopy i.e. 'catastrophic regex'.

---

---

> and it saids safari cannot open page for its invalid on my iphone
>
> Also this link you keep sending

if you mean error 404 at `fiddle.net` then check https://jsfiddle.net/9ngLtwx5/2/

if you mean error about:
> "Refused to display '`https://www3.zoechip.com/`' in a frame because it set 'X-Frame-Options' to 'sameorigin'." 

then may needed is found similar WebExtension for iOS: 

https://github.com/ThomazPom/Moz-Ext-Ignore-X-Frame-Options
https://github.com/guilryder/chrome-extensions/tree/main/xframe_ignore

Unfortunately, I do not know for iOS such yet.

---

> Why is everytime i report miss ads on the adguard website and create a ticket that ticket is just added into the previous ones i also made on the same thread?

As you use the tool built into the AdGuard application what redirects to the dedicated online form "`https://reports.adguard.com/en/new_issue.html`", it is the expected behavior that the AdGuard Bot will paste this into the open submission as a comment.

As for this topic of yours:

>![image](https://github.com/AdguardTeam/AdguardFilters/assets/35370833/3eab5441-fb36-48c3-bf58-c6a8905cf02a)

 maybe you don't understand the difference between the "create new issue" button and the "send comment" button.

**Comment by olstice** [comment]
So did we resolve the IOS issue with https://www3.zoechip.com/?

**Comment by olstice** [comment]
> > all this did is showed the inspect element html of the website without me needing to go into inspect element
> 
> There, to me, it's just about fooling the server that you are visiting a site for the first time on a given IP (some fraction of malware checks the IP to avoid loading a second time on the site).
> 
> > Krystian I dont know how to create my own filter rules i dont understand any of this i tried to and i dont get it so i need someone to do this for me, because im not capable
> 
> OK, if you have the desire you can try to learn what a wildcard is from other sources such as Adblock Plus specifications:
> 
> https://forum.adblockplus.org/viewtopic.php?t=54688
> 
> https://help.adblockplus.org/hc/en-us/articles/360062733293#anchors
> 
> "In short, an asterisk in a simple web filter replaces 1 or many characters - they allow any combination."
> 
> Mastering regexes rather well start with some beginner's courses and perhaps you will refresh it to yourself throughout your life e.g. problems with writing regexes so that it is not too loopy i.e. 'catastrophic regex'.
> 
> > and it saids safari cannot open page for its invalid on my iphone
> > Also this link you keep sending
> 
> if you mean error 404 at `fiddle.net` then check https://jsfiddle.net/9ngLtwx5/2/
> 
> if you mean error about:
> 
> > "Refused to display '`https://www3.zoechip.com/`' in a frame because it set 'X-Frame-Options' to 'sameorigin'."
> 
> then may needed is found similar WebExtension for iOS:
> 
> https://github.com/ThomazPom/Moz-Ext-Ignore-X-Frame-Options https://github.com/guilryder/chrome-extensions/tree/main/xframe_ignore
> 
> Unfortunately, I do not know for iOS such yet.
> 
> _Here, I'll agree that it was my two mistakes (that I didn't check that the page defends against loading into an iframe/object/embed and that I didn't check that the link to the fullscreen version, however, expires after time hence the internal 404 error)._
> 
> > Why is everytime i report miss ads on the adguard website and create a ticket that ticket is just added into the previous ones i also made on the same thread?
> 
> As you use the tool built into the AdGuard application what redirects to the dedicated online form "`https://reports.adguard.com/en/new_issue.html`", it is the expected behavior that the AdGuard Bot will paste this into the open submission as a comment.
> 
> As for this topic of yours:
> 
> > ![image](https://user-images.githubusercontent.com/35370833/259012064-3eab5441-fb36-48c3-bf58-c6a8905cf02a.png)
> 
> maybe you don't understand the difference between the "create new issue" button and the "send comment" button.

Again I barely understand anything your saying

So please do not ask me to create a userrule or an ad filter I dont comprehend this


Please just get this resolved for me

**Comment by olstice** [comment]
> > all this did is showed the inspect element html of the website without me needing to go into inspect element
> 
> There, to me, it's just about fooling the server that you are visiting a site for the first time on a given IP (some fraction of malware checks the IP to avoid loading a second time on the site).
> 
> > Krystian I dont know how to create my own filter rules i dont understand any of this i tried to and i dont get it so i need someone to do this for me, because im not capable
> 
> OK, if you have the desire you can try to learn what a wildcard is from other sources such as Adblock Plus specifications:
> 
> https://forum.adblockplus.org/viewtopic.php?t=54688
> 
> https://help.adblockplus.org/hc/en-us/articles/360062733293#anchors
> 
> "In short, an asterisk in a simple web filter replaces 1 or many characters - they allow any combination."
> 
> Mastering regexes rather well start with some beginner's courses and perhaps you will refresh it to yourself throughout your life e.g. problems with writing regexes so that it is not too loopy i.e. 'catastrophic regex'.
> 
> > and it saids safari cannot open page for its invalid on my iphone
> > Also this link you keep sending
> 
> if you mean error 404 at `fiddle.net` then check https://jsfiddle.net/9ngLtwx5/2/
> 
> if you mean error about:
> 
> > "Refused to display '`https://www3.zoechip.com/`' in a frame because it set 'X-Frame-Options' to 'sameorigin'."
> 
> then may needed is found similar WebExtension for iOS:
> 
> https://github.com/ThomazPom/Moz-Ext-Ignore-X-Frame-Options https://github.com/guilryder/chrome-extensions/tree/main/xframe_ignore
> 
> Unfortunately, I do not know for iOS such yet.
> 
> _Here, I'll agree that it was my two mistakes (that I didn't check that the page defends against loading into an iframe/object/embed and that I didn't check that the link to the fullscreen version, however, expires after time hence the internal 404 error)._
> 
> > Why is everytime i report miss ads on the adguard website and create a ticket that ticket is just added into the previous ones i also made on the same thread?
> 
> As you use the tool built into the AdGuard application what redirects to the dedicated online form "`https://reports.adguard.com/en/new_issue.html`", it is the expected behavior that the AdGuard Bot will paste this into the open submission as a comment.
> 
> As for this topic of yours:
> 
> > ![image](https://user-images.githubusercontent.com/35370833/259012064-3eab5441-fb36-48c3-bf58-c6a8905cf02a.png)
> 
> maybe you don't understand the difference between the "create new issue" button and the "send comment" button.

please stop trying to explain this all to me i dont comprehend it at all

just use words everyone understands of the problem is being looked at or its been resolved

i tried to get this and i dont so iv accepted i am just average and basic and i do not comprehend this

**Comment by krystian3w** [comment]
Rather, the problem is not being analyzed or considered for repair.

---

In addition, you quote the whole thing and I answered you nicely on one of the questions:

> Why is everytime i report miss ads on the adguard website and create a ticket that ticket is just added into the previous ones i also made on the same thread?

* As you use the tool built into the AdGuard application what redirects to the dedicated online form "`https://reports.adguard.com/en/new_issue.html`", it is the expected behavior that the AdGuard Bot will paste this into the open submission as a comment.

* As for this topic of yours:

  >![image](https://github.com/AdguardTeam/AdguardFilters/assets/35370833/3eab5441-fb36-48c3-bf58-c6a8905cf02a)

 "then" maybe you don't understand the difference between the "create new issue" button and the "send comment" button.

*So either you are not a native of English.*

**Comment by olstice** [comment]
> Rather, the problem is not being analyzed or considered for repair.
> 
> In addition, you quote the whole thing and I answered you nicely on one of the questions:
> 
> > Why is everytime i report miss ads on the adguard website and create a ticket that ticket is just added into the previous ones i also made on the same thread?
> 
>     * As you use the tool built into the AdGuard application what redirects to the dedicated online form "`https://reports.adguard.com/en/new_issue.html`", it is the expected behavior that the AdGuard Bot will paste this into the open submission as a comment.
> 
>     * As for this topic of yours:
>       > ![image](https://user-images.githubusercontent.com/35370833/259012064-3eab5441-fb36-48c3-bf58-c6a8905cf02a.png)
> 
> 
> "then" maybe you don't understand the difference between the "create new issue" button and the "send comment" button.
> 
> _So either you are not a native of English._

Why is the problem not being analyzed or repaired? 

Also i quoated because i dont know how to ping you on the comment, 


also i am a native of english, english is my only language 


Also why are you being rude to me?

maybe i am dumb hence your "so either you are not a native of english"

**Comment by olstice** [comment]
@krystian3w I did not know yet until just now how to address a specific person which is why i was doing the quotes now that i know i wont do them anymore. 

but i do want to know why you said the problem was not going to be analyzed or repaired?

also i am a native of english, english is my only language

Also why are you being rude to me?

maybe i am dumb hence your "so either you are not a native of english"
